robottelo/host_helpers/__init__.py

```diff
@@ -8,5 +9,9 @@ class ContentHostMixins(SystemFacts, VersionedContent):
     pass
 
 
-class SatelliteMixins(ContentInfo, Factories):
+class CapsuleMixins(CapsuleInfo, ContentHostMixins):
```

Comment:
No need to include ContentHostMixins here since ContentHost is a base class of Capsule.